docs/product-getting-started-guide/src/main/asciidoc/data-object-violation-create-proc.adoc

```diff
@@ -0,0 +1,79 @@
+[id='data-object-violation-proc']
+= Creating the Violation data object
+
+The following chapter instructs you how to create the Violation data object containing data fields, such as: Violation Date, Fine Amount, and Speed Limit.
+
+[float]
+== Prerequisites
+
+You have created the Driver department traffic violations project.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Data Object*.
+. In the *Create new Data Object* wizard, enter the following values:
+.. *Data Object*: `Violation`.
```

Comment:
These need to be bullets, not arabic numerals, which are reserved for sequential sub-steps (though avoided if possible).

Reply:
Good catch. Done